tasks/databases.yml

```diff
@@ -5,4 +5,5 @@
     collation: "{{ item.collation | default('utf8_general_ci') }}"
     encoding: "{{ item.encoding | default('utf8') }}"
     state: present
+  sudo: no
```

Comment:
We also use another user, but we don't log in as that user with ansible, so "sudo: no" would fail in our case.